create_framework/http-foundation.rst

```diff
@@ -121,6 +121,15 @@ To use this component, add it as a dependency of the project:
 
 Running this command will also automatically download the Symfony
 HttpFoundation component and install it under the ``vendor/`` directory.
+A ``composer.json`` and a ``composer.lock`` file will be generated as well:
+
+.. code-block:: json
+
+    // composer.json
```

Comment:
Well, I don't consider this as an issue with the syntax highlighting. It's just that someone will probably copy the code example and will then run into syntax errors because the example itself is no valid JSON.

I would rather change the example and maybe write an explanation below the code.